### PR TITLE
Adding Material Dashboard bootstrap template

### PR DESCRIPTION
**Pull request by lotif**
# PR Type
Feature

# Short Description

Clickup Ticket(s): https://app.clickup.com/t/86877ahk0

Creating a basic template based on https://www.creative-tim.com/product/material-dashboard#

Currently looking like this:

Screenshot 2024-02-01 at 16 45 40

# Tests Added
NA

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                    <li class="nav-item">
+                        <a class="nav-link text-white active bg-gradient-primary" href="index.html">
+                            <div class="text-white text-center me-2 d-flex align-items-center justify-content-center">
+                                <i class="material-icons opacity-10">home</i>
+                            </div>
+                            <span class="nav-link-text ms-1">Home</span>
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-white " href="about.html">
```
Comment (jewelltaylor) [inline index.html:59]: Should we include a blank about.html page or take out this ref since it currently doesn't exist?

**Review by jewelltaylor — APPROVED** [review]
Other than the small comment I made, I think this looks good to me as the initial UI template! I made some tickets in clickup as some follows ups that seem like logical next steps to take on for us:
- [Port Project Template to Next.js front end](https://app.clickup.com/t/86879c2bk)
- [Figure out static checks for JavaScript or TypeScript portion of the app](https://app.clickup.com/t/86879c71k)

Feel free to modify these or change them if you do not agree :)

File: index.html (hunk 1 above)
Comment (lotif) [reply index.html:59]: I added it more as a reference for having another manu item, but I'll remove to avoid confusion :)

**Comment by lotif** [comment]
I had a ticket for the first one already (https://app.clickup.com/t/86877agzf) so I'll close it, but the second one is a great idea. Thanks!

File: index.html (hunk 1 above)
Comment (jewelltaylor) [reply index.html:59]: Yeah I figured hahah no worries in that case :)

**Comment by jewelltaylor** [comment]
> I had a ticket for the first one already (https://app.clickup.com/t/86877agzf) so I'll close it, but the second one is a great idea. Thanks!

My bad I missed that one cause you already had it in progress!

**Pull request merged**